config/runtime.exs

```diff
@@ -91,10 +91,29 @@ default_locale =
 config :gettext, :default_locale, default_locale
 config :timex, :default_locale, default_locale
 
-# enable brainstorming teasers:
+delete_brainstormings_after_days =
+  if System.get_env("MW_FEATURE_BRAINSTORMING_REMOVAL_AFTER_DAYS") &&
+       System.get_env("MW_FEATURE_BRAINSTORMING_REMOVAL_AFTER_DAYS") != "",
+     do: String.to_integer(System.get_env("MW_FEATURE_BRAINSTORMING_REMOVAL_AFTER_DAYS")),
+     else: 30
+
+# enable/disable brainstorming teasers and configure delete brainstormings option:
 config :mindwendel, :options,
   feature_brainstorming_teasers:
     Enum.member?(
       ["", "true"],
       String.trim(System.get_env("MW_FEATURE_BRAINSTORMING_TEASER") || "")
-    )
+    ),
+  feature_delete_brainstormings_after_days: delete_brainstormings_after_days
+
+if config_env() == :prod || config_env() == :dev do
```

Comment:
The following remark is not part of this pull request. But we need to put this config in other files, e.g. `runtime_dev.exs` and `runtime_prod.exs` .

Reply:
at least we should learn about it and best practices, true